catalog/app/utils/PackageUri.ts

```diff
@@ -108,5 +108,6 @@ export function stringify({ bucket, name, hash, tag, path }: PackageUri) {
     pkgSpec += `:${tag}`
   }
   const pathPart = path ? `&path=${encodeURIComponent(path)}` : ''
-  return `quilt+s3://${bucket}#package=${pkgSpec}${pathPart}`
+  const catalogPart = window.location.hostname
```

Comment:
even tho it may seem like a nice shortcut, i deeply disagree with this approach.
i believe the `catalog` part should be added to the model and handled explicitly

Reply:
Okay, I've interpreted `model` as "stringify takes it as an argument", which is passed from:
- Dashboard
- PackageTreeWrapper (on through to PackageCodeSample)

Is that what you meant? If not, any chance you could fix it for me so it makes this release?

Reply:
> Is that what you meant?

yep

> If not, any chance you could fix it for me so it makes this release?

ok ill try